### PR TITLE
Supports Unemojify. 

### PR DESCRIPTION
**Pull request by shivkanthb**
Usage:
```
emoji.unemojify("I ❤️  ☕️! -  😯⭐️😍  ::: test : : 👍+");
```
Outputs:
```
I :heart:  :coffee:! -  :hushed::star::heart_eyes:  ::: test : : :+1:+
```
Fixes issue #36

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -5,3 +5,4 @@ node_modules
 *.txt
 test.js
 coverage
+play.js
```
Comment (omnidan) [inline .gitignore]: why is this added to the gitignore file? if you just used this file to test out the library, please remove it from the pull request.

File: lib/emoji.js [hunk 2]
```diff
@@ -1,5 +1,6 @@
 /*jslint node: true*/
 require('string.prototype.codepointat');
+var toArray = require('lodash/toArray');
```
Comment (omnidan) [inline lib/emoji.js]: I don't really want to pull in all of lodash as a dependency just for a way to convert something to an array - surely there must be a way to accomplish the same behaviour without lodash?

File: lib/emoji.js [hunk 3]
```diff
@@ -126,3 +127,30 @@ Emoji.search = function search(str) {
     };
   });
 }
+
+Object.prototype.getKeyByValue = function(value) {
+  for(var prop in this) {
```
Comment (omnidan) [inline lib/emoji.js]: please stick to the style in the rest of the library (space after `for` and `if`)

File: test/emoji.js [hunk 4]
```diff
@@ -110,4 +110,24 @@ describe("emoji.js", function () {
...
+    it("should parse emoji and replace them with :emoji:", function() {
+      var coffee = emoji.unemojify('I ❤️  ☕️! -  😯⭐️😍  ::: test : : 👍+');
+      should.exist(coffee);
+      coffee.should.be.exactly('I :heart:  :coffee:! -  :hushed::star::heart_eyes:  ::: test : : :+1:+');
+    })
+
+    it("should leave unknown emoji", function () {
+      var coffee = emoji.unemojify('I ⭐️ :another_one: 🥕');
+      should.exist(coffee);
+      coffee.should.be.exactly('I :star: :another_one: 🥕');
```
Comment (omnidan) [inline test/emoji.js]: shouldn't this un-emojify the carrot emoji?

File: lib/emoji.js (hunk 3 above)
Comment (shivkanthb) [inline lib/emoji.js]: Oops. Missed that. Will do! 👍

File: test/emoji.js (hunk 4 above)
Comment (shivkanthb) [inline test/emoji.js]: Right now it only unemojifies the ones present in emoji.json. If carrot is added to that, this will be resolved automatically.

File: .gitignore (hunk 1 above)
Comment (shivkanthb) [inline .gitignore]: Pushed this by mistake. Will remove 👍

File: lib/emoji.js (hunk 2 above)
Comment (shivkanthb) [inline lib/emoji.js]: So the issue is with some complex emojis. 
let's take an example. 
```
var str = '👩‍❤️‍💋‍👩';

/* using lodash's toArray */
toArray(str);  //  outputs [ '👩‍❤️‍💋‍👩' ] (length: 1)

/* using Array.from() */
Array.from(str);  //  outputs: [ '👩', '‍', '❤', '️', '‍', '💋', '‍', '👩' ] (length : 8)

/* using string split() */
str.split('');  //  outputs [ '�', '�', '‍', '❤', '️', '‍', '�', '�', '‍', '�', '�' ] (length : 11)
```
I searched quite a bit to achieve this without lodash but no luck 👎 .

Comment (omnidan) [inline lib/emoji.js]: @shivkanthb I see, in that case, it might be best to just use https://www.npmjs.com/package/lodash.toarray (so that we don't have to have all of lodash as dependency)

File: test/emoji.js (hunk 4 above)
Comment (omnidan) [inline test/emoji.js]: Ah, I see, that makes sense.

**Comment by omnidan** [comment]
@shivkanthb did you manage to review the comments and push your changes yet?

File: lib/emoji.js (hunk 2 above)
Comment (shivkanthb) [inline lib/emoji.js]: Updated 👍

File: lib/emoji.js (hunk 3 above)
Comment (shivkanthb) [inline lib/emoji.js]: Updated :)

**Comment by omnidan** [comment]
@shivkanthb thanks! can you also document this feature in the README.md?

**Comment by shivkanthb** [comment]
Readme updated 👍

**Comment by omnidan** [comment]
@shivkanthb looks good now - thank you so much :grin: :ok_hand:

**Comment by omnidan** [comment]
released `1.6.0` with this PR merged

**Comment by omnidan** [comment]
@shivkanthb actually, this PR ended up breaking yarn (#45 and yarnpkg/yarn#3851), so I had to revert it. Someone came up with a possible fix in #46
